src/crush/CrushWrapper.h

```diff
@@ -233,6 +238,24 @@ class CrushWrapper {
     crush->straw_calc_version = n;
   }
 
+  int get_msr_descents() const {
+    return crush->msr_descents;
+  }
+  void set_msr_descents(int n) {
```

Comment:
Is `int` the intended type here? `struct crush_map` defines them both as `__u8`.

Reply:
Swapped to unsigned.